app/src/main/java/com/example/kotlinandroidhelloworld/MainActivity.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.kotlinandroidhelloworld
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.widget.TextView
+
+/**
+ * Let's convert this class to Kotlin as an example!
+ */
+class MainActivity : AppCompatActivity() {
+
+    public override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        var str = "Here is some more text\n"
+
+        str += "$(str.toString)";
```

Comment:
No semi-colons needed :) 

This reminds me I didn't cover string substitution in our little intro, but it'll probably come up many more times:
```kotlin
val num = 5
val str = "Your number ($num) plus 2 is: ${num + 2}"
```